spec/forms/developer_search_form.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DeveloperSearchForm do
+  describe '#search' do
+
+    let(:language) { create :language }
+    let(:programming_language) { create :programming_language }
+
+    before do
+      create(:developer, programming_languages: [programming_language], languages: [language])
+      create :developer
+    end
+
+    context 'with no params' do
+      it 'returns all users' do
+        developer_search_form = DeveloperSearchForm.new
+        expect( developer_search_form.search.size).to eq(2)
```

Comment:
I wonder why you put an unnecessary space randomly.
See: https://github.com/bbatsov/ruby-style-guide#spaces-braces

Reply:
ahhh. It's my mistake. I trying to change variable use find and replace feature in Editor. And it redundant space :|
Fixed!

Reply:
Alright.
Then please check the code on github before asking for review.